package.json

```diff
@@ -115,8 +118,9 @@
     "start-docs": "npm run build-min && npm run build-docs && jekyll serve --watch",
     "lint": "eslint --cache --ignore-path .gitignore src test bench docs/_posts/examples/*.html debug/*.html",
     "lint-docs": "documentation lint src/index.js",
+    "lint-css": "stylelint 'docs/*.css' || true",
```

Comment:
I think you should be pointing the linter at `dist/mapbox-gl.css` and not the `docs/` css files. We only enforce the `mapboxgl-` prefex for css classes that ship with the library distribution. The CSS in the `docs/` folder is only for the mapbox documentation website - www.mapbox.com/mapbox-gl-js/api 
That change might also remove the need for the `|| true` statement, although I'm not 100% clear on what that is doing in this case.